README.md

```diff
@@ -168,7 +173,7 @@ The best way to fully validate your setup is to run automated tests with `pytest
 1. Make sure you have Python 3.8+ as the default python in your environment. Using _virtualenv_ or _conda_, etc. for isolation is of course encouraged.
 1. Install the package from local sources: `pip install -e .`.
 1. Install additional packages required for tests: `pip install -r test-requirements.txt`
-1. Run `./test-docker-compose.sh`. This script takes care to set environment variables for tests to connect to the running docker-compose services.
+1. Run `pytest -vv`.
```

Comment:
I've got some failing tests, is that my fault?
![image](https://user-images.githubusercontent.com/23058747/113660275-651ef100-96ac-11eb-8312-f9d6bc10b730.png)